dev/NavigationView/NavigationViewItemPresenter.h

```diff
@@ -23,8 +23,12 @@ class NavigationViewItemPresenter:
 
     winrt::UIElement GetSelectionIndicator();
 
+    void UpdateCompactPaneLength(double compactPaneLength,bool shouldUpdate);
+
 private:
     NavigationViewItem * GetNavigationViewItem();
 
+    double m_compactPaneLengthValue = 40;
```

Comment:
>double m_compactPaneLengthValue = 40; [](start = 4, length = 37)

I think the default vaule of this should probably be a theme resource?

Reply:
Currently the compact pane length is a property of the NavigationView, so I am not sure how a theme resource for that would go with that.

Reply:
So doesn't that mean if we change compact pane length on NavigationView this value would get out of sync?

Reply:
When the compact pane length of the parent NavigationView changes, this gets also passed to the NavigationViewItem which then calls the NavigationViewItemPresenter.UpdateCompactPaneLength function.